profile/src/test/java/com/scottlogic/deg/profile/v0_1/ProfileSchemaValidatorTests.java

```diff
@@ -63,20 +59,20 @@ private File getFileFromURL(String profileDirName) {
         File[] listOfFiles = getFileFromURL(INVALID_PROFILE_DIR).listFiles(jsonFilter);
         Collection<DynamicTest> dynTsts = new ArrayList<>();
 
-        for (int i = 0; i < listOfFiles.length; i++) {
-            String profileFilename = listOfFiles[i].getName();
-            DynamicTest test = DynamicTest.dynamicTest(profileFilename, () -> {
+        for (File file : listOfFiles) {
```

Comment:
I would avoid naming this `listOfFiles` if it's actually an array, to avoid confusion with the `List<T>` class.